docs/UserGuide.md

```diff
@@ -41,25 +44,44 @@ title: User Guide
    Some example commands you can try:
 
 
-   * **`add`**`n/John Doe p/98765432 l/Chinese a/John street, block 123, #01-01` : Adds an elderly contact named `John Doe` to the SeniorLove.
+   * `add n/John Doe p/98765432 l/Chinese a/John street, block 123, #01-01` : Adds an elderly contact named `John Doe` to the SeniorLove.
 
-   * **`delete`**`3` : Deletes the 3rd elderly shown in the current list.
+   * `delete 3` : Deletes the 3rd elderly shown in the current list.
 
-   * **`list`** : Lists all elderly.
+   * `list` : Lists all elderly.
 
-   * **`find`**`Hans` : Find all elderly whose attributes start with Hans.(Case insensitive)
+   * `find Hans` : Find all elderly whose attributes start with Hans.(Case insensitive)
 
-   * **`clear`** : Deletes all elderly.
+   * `clear` : Deletes all elderly.
 
-   * **`exit`** : Exits the app.
+   * `exit` : Exits the app.
 
 7. Refer to the [Features](#features) below for details of each command.
 
+--------------------------------------------------------------------------------------------------------------------
+## Glossary
+
+This section provides information on commonly used terms throughout SeniorLove.
+
+### Terminology used
+
+Term            | Meaning
+----------------|-----------------
+Elderly | A person in SeniorLove
+Attribute | Data that is assigned to an elderly
+
```

Comment:
Might want to add Visit as a term as well, since it is used in the next section of structure of elderly contact